### ISSUE TITLE
Rename to glacier-3dviz?

### ISSUE THREAD

**Issue by fmaussion** [OGGM/glacier3dviz#21]
I think I'd like to keep the repo in the OGGM organisation namespace (since we seeked funding for it) but we can rename it to something more general

**Comment by pat-schmitt** [comment]
Yes, I just wanted to start with the renaming today. I thought about glacier3dviz with out the minus

**Comment by pat-schmitt** [comment]
Maybe we can but the tutorials also on a website like glacier3dviz.oggm.org?

**Comment by fmaussion** [comment]
> Maybe we can but the tutorials also on a website like glacier3dviz.oggm.org?

Yes getting a subdomain is easy (we just have to ask Ben). It's all about getting a github workflows that works, or building locally and hosting on ghpages. Do you have a jupyterbook setup?

**Comment by pat-schmitt** [comment]
No not now. I wanted to copy the setup from the tutorials and adapt it. And building locally should be enough for starting.
